ukvm/ukvm.h

```diff
@@ -142,4 +142,18 @@ extern struct ukvm_module ukvm_module_blk;
 extern struct ukvm_module ukvm_module_net;
 extern struct ukvm_module ukvm_module_gdb;
 
+/*
+ * GDB specific functions to be implemented on all backends for all
+ * architectures.
+ */
```

Comment:
Could you document these functions similarly to how the other public functions are documented in this file?

Also, please use `ukvm_gpa_t` for addresses (as opposed to `uint64_t`) and `size_t` for the various `len` parameters which are a byte size (as opposed to `uint32_t` etc).